react-frontend/src/components/StyleComponents/bannerSimple.js

```diff
@@ -45,11 +44,11 @@ export const BannerTitle = styled.h1.attrs({
   className: 'bannerTitle',
 })`
   color: #313132;
+  font-family: ‘BC Sans’, ‘Noto Sans’, Verdana, Arial, sans-serif;
   font-size: 37px;
   font-weight: bold;
-  line-height: 37px;
+  line-height: 1.2;
```

Comment:
To ensure titles that break across lines aren't too crunched together. For example:

Before:
![Screen Shot 2020-10-26 at 12 58 54 PM](https://user-images.githubusercontent.com/16946297/97222180-30b51980-178b-11eb-839e-b476d0ed1868.png)


After:
![Screen Shot 2020-10-26 at 12 59 17 PM](https://user-images.githubusercontent.com/16946297/97222190-36126400-178b-11eb-9be1-9badc33ee519.png)